src/main/resources/openapi/transcriptions.yaml

```diff
@@ -585,6 +617,45 @@ components:
         requested_ts:
           type: string
           format: date-time
+    TranscriberViewSummary:
+      type: object
+      required:
+        - transcription_id
+        - case_id
+        - case_number
+        - courthouse_name
+        - hearing_date
+        - transcription_type
+        - status
+        - requested_ts
+        - state_change_ts
+        - is_manual
+      properties:
```

Comment:
Can any tighter validation be applied to the `string` types, e.g `maxLength`?

Reply:
I'm going to leave this one as is - it's just a summary view object in the OK response.
We don't have any limitations on string length in our database tables.